pkgs/development/r-modules/default.nix

```diff
@@ -275,7 +276,8 @@ let
     jqr = [ pkgs.jq.dev ];
     KFKSDS = [ pkgs.gsl_1 ];
     kza = [ pkgs.fftw.dev ];
-    lwgeom = [ pkgs.gdal pkgs.geos pkgs.proj ];
+    lpsymphony = [ pkgs.pkgconfig pkgs.gfortran pkgs.gettext ];
+    lwgeom = [pkgs.proj.dev pkgs.proj pkgs.pkg-config pkgs.geos pkgs.sqlite pkgs.gdal ];
```

Comment:
```suggestion
    lpsymphony = with pkgs; [ pkg-config gfortran gettext ];
    lwgeom = with pkgs; [ proj.dev proj pkg-config geos sqlite gdal ];
```

Reply:
I have fixed the repeated `pkgs.` everywhere it came up in `r-modules/default.nix`